src/itest/kotlin/nl/info/zac/itest/ZaakKoppelenRestServiceTest.kt

```diff
@@ -64,41 +66,47 @@ class ZaakKoppelenRestServiceTest : BehaviorSpec({
                       "identificatie": "ZAAK-2000-0000000006",
                       "isKoppelbaar": false,
                       "omschrijving": "fakeOmschrijving",
+                      "zaaktypeOmschrijving": "${ZAAKTYPE_INDIENEN_AANSPRAKELIJKSTELLING_DOOR_DERDEN_BEHANDELEN_DESCRIPTION}",
```

Comment:
```suggestion
                      "zaaktypeOmschrijving": "$ZAAKTYPE_INDIENEN_AANSPRAKELIJKSTELLING_DOOR_DERDEN_BEHANDELEN_DESCRIPTION",
```

IntelliJ should warn you of this? I thought even Detekt would have failed on this but apparently not.